apps/asap-server/src/utils/sync-google-event.ts

```diff
@@ -1,10 +1,61 @@
-import Joi from '@hapi/joi';
-import { RestEvent, Event } from '@asap-hub/squidex';
+import { Event, RestEvent } from '@asap-hub/squidex';
 import { EventStatus } from '@asap-hub/model';
 import { calendar_v3 as calendarV3 } from 'googleapis';
+import { JSONSchemaType } from 'ajv';
 import { EventController } from '../controllers/events';
 import logger from './logger';
-import { RequiredAndNonNullable } from './squidex';
+import { validateInput } from '../validation';
+import { NullableOptionalProperties } from './types';
+
+type GoogleEvent = NullableOptionalProperties<{
```

Comment:
@peterstarling should we move all validation schemas into a `.validation.ts` file? Even if they are used only once?

Reply:
I guess we could? why not

Reply:
will do (in the other PR)